src/test/java/api/cart/PostCartTests.java

```diff
@@ -37,4 +38,13 @@ public void addCartBillingAddress() {
         Cart postCartBillingAddress = cartApiWrapper.postCartBillingAddress(billingAddress, postCartResponse.getId());
         Assert.assertEquals(cartApiWrapper.statusCode, HttpStatus.SC_OK, "incorrect error code");
     }
+
+    @Test
+    public void addCustomerInfoToCart() {
+        Cart cart = new Cart();
+        Cart postCartResponse = cartApiWrapper.postEmptycart(cart);
+        Customer infoToAdd = CartUtils.addCustomerInfoToCart("Gelu", "Haiduc", "haiduc.gelu@email.com");
+        Cart postCartCustomerInfo = cartApiWrapper.postCartCustomerInfo(infoToAdd, postCartResponse.getId());
+        Assert.assertEquals(cartApiWrapper.statusCode, HttpStatus.SC_OK, "incorrect error code");
```

Comment:
Test looks good, but besides the response status, we should also check that the correct customer info is returned.
Also, we should be more specific on assertion errors: instead of "incorrect error code", "Incorrect status code when adding customer info to cart"